### PR TITLE
Style core elements

### PR DESCRIPTION
**Pull request by disc3110**
Add style to the page using bootstrap

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,20 +6,31 @@
...
+        <div class="card-body">
+          <h4 class="card-title text-center mb-3">Add a new Book</h4>
+          <input type="text" class="form-control my-2" placeholder="Title" id="new-book-title">
+          <input type="text" class="form-control my-2" placeholder="Author" id="new-book-author">
+          <div class="text-end">
+            <button type="button" class="add-btn btn btn-primary mt-3"> Add </button>
+          </div>
+        </div>
+      </form>
     </div>
```
Comment (umararshadbut) [inline index.html:33]: Make sure that this section is align-center according to the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png). And make sure that the book title, writer name and remove button should be in the same row.

**Review by umararshadbut — CHANGES_REQUESTED** [review]
Hi @disc3110 and @brenoxav , 

Good job so far :+1:!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
HAPPY CODING :computer:
**Umair Arshad**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by brenoxav** [comment]
Hello, @umararshadbut!
Thank you for your review.

The link to the live demo was wrong.
We have already fixed that.

Please, [check the correct link.](http://brenoxavier.com/awesome-books/)

**Review by voscarmv — APPROVED** [review]
Hi @disc3110 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**